src/components/downloadModal.tsx

```diff
@@ -181,6 +181,13 @@ export const DownloadModal: React.FC<{
         </section>
 
         <footer className="modal-card-foot is-justify-content-flex-end">
+          <a
+            className="button is-normal is-rounded"
+            href="https://voicevox.hiroshiba.jp/term/"
+            target="_blank"
+          >
+            <span>VOICEVOX 利用規約</span>
+          </a>
```

Comment:
* ページ内遷移は`Link`だと再描画が走らないので早いのでそちらに変更
* デザインをちょこっと調整

```suggestion
          <Link to="/term/" className="button">
            <span>利用規約</span>
          </Link>
```